dotCMS/src/integration-test/resources/it-dotmarketing-config.properties

```diff
@@ -317,7 +317,7 @@ path.to.dojo=/html/js/dojo/custom-build
 URLMAP_FALLTHROUGH=true
 
 ##	Reserved Folder Names
-RESERVEDFOLDERNAMES=WEB-INF,META-INF,assets,dotcms,html,portal,email_backups,DOTLESS,DOTSASS
+RESERVEDFOLDERNAMES=WEB-INF,META-INF,assets,dotcms,html,portal,email_backups,DOTLESS,DOTSASS,dotAdmin,custom_elements
```

Comment:
remove this property from the properties file

Reply:
done